solutions/2300-2399/2302-count-subarrays-with-score-less-than-k.md

```diff
@@ -0,0 +1,119 @@
+---
+description: >-
+  Author: @saloni33 |
+  https://leetcode.com/problems/count-subarrays-with-score-less-than-k/
+---
+
+# 2302 - Count Subarrays With Score Less Than K (Hard)
+
+## Problem Link
+
+https://leetcode.com/problems/count-subarrays-with-score-less-than-k/
+
+## Problem Statement
+
+The $score$ of an array is defined as the $product$ of its sum and its length.
+
+For example, the score of [1, 2, 3, 4, 5] is (1 + 2 + 3 + 4 + 5) * 5 = 75.
```

Comment:
Use LaTeX for numbers - $[1, 2, 3, 4, 5]$

Use bulletin for this line as per LC